### PR TITLE
Debian 11 HT web VM

### PR DESCRIPTION
**Pull request by aelkiss**
Per discussion in Slack, and not wanting to make a PR that (if merged) will break a bunch of stuff, I've made this against a new bullseye branch.

### CODE REVIEW

**Comment by aelkiss** [comment]
Tests are currently failing because of a conflict between requiring `mariadb-client` in `manifests/profile/hathitrust/dependencies.pp` and something the chipmunk role is bringing in. I think using `ensure_packages` should fix it.

**Comment by aelkiss** [comment]
Assuming we merge this to the `bullseye` branch, we should follow this up with a change to:

- add a puppet.conf on ae-files that defaults to the bullseye environment
- update https://github.com/mlibrary/nebula/blob/7166267a24b1c1acfbc5b8a88b4d41879968f79f/templates/virtual_machine/bullseye.cfg.erb#L486 to use that puppet.conf by default

**Comment by aelkiss** [comment]
@skorner  says:

> have you noticed the logrotate failure emails the past week or so for your new VMs? The common issue is a duplicate config for `btmp` and `wtmp` (both seem to come from puppet based on the conf.d file header, but one prefixed with `debian_` ). The Fulcrum VMs have a separate issue wrt the perms on the app log path.

File: spec/classes/role/htvm_test_webhost_spec.rb [hunk 1]
```diff
@@ -9,8 +9,9 @@
 describe 'nebula::role::webhost::htvm::test' do
   on_supported_os.each do |os, os_facts|
     context "on #{os}" do
+      require 'pry'
```
Comment (aelkiss) [inline spec/classes/role/htvm_test_webhost_spec.rb:12]: need to remove this

**Review by rrotter — COMMENTED** [review]
All looks fine to me style-wise. Before we merge this I'd like to hear from @daaang on the dependency updates so we don't conflict with upstream.

**Comment by aelkiss** [comment]
Next step is to get this to a point where as much as possible except for incompatible php module updates can be merged. Will make a fresh branch/PR for that work.

**Comment by aelkiss** [comment]
Superceded by #512 , #513

**Pull request closed without merging**